exercises/practice/acronym/test.8th

```diff
@@ -5,50 +5,50 @@ with: test
 9 tests
 
 "ignoring punctuation and casing"
-    ( "Complementary metal-oxide semiconductor" acronym )
     "CMOS"
+    ( "Complementary metal-oxide semiconductor" acronym )
     equal?
```

Comment:
In the proposed change, `drop dup` would be a valid implementation of `acronym`.

Reply:
As discussed in https://forum.exercism.org/t/i-suggest-updating-few-8th-tests-to-not-allow-hack-shortcut/13439, the student's answer hits the stack before the expected value.